var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -35,6 +35,7 @@ class Trilinos(CMakePackage):
     version('xsdk-0.2.0', tag='xsdk-0.2.0')
     version('develop', branch='develop')
     version('master', branch='master')
+    version('12.14.0-rc1', commit='dbf41f3b26b0326a1377f219e6e07eab861d181e')  # branch trilinos-release-12-14-branch
```

Comment:
I don't normally make release candidates the default version of a package. I would set `preferred=True` on the last stable release.